script/Arcueid/Day 5/05_00_ARC05_9_2.txt

```diff
@@ -7,52 +7,53 @@ It has to be gliding.
 [sha:eefe338a71ab6b56ff5e167521f134a9cd7de89c]{
 -- Page 10021, Offset 10384.
 -- いや、正しくは落下に近い。
-No, more accurately closer to falling.
+Or rather, something more akin to falling.
 }
 [sha:ec152f17a92ded8286eb9b792d3ca1f19305b1b4]{
 -- Page 10021, Offset 10385.
 -- ……だができるか？　たどり着けるかは運任せだ。
-But can I do it...? It's up to fate whether or not I make it.
+But can I do it? Whether I make it or not would be up to chance.
 }
 [sha:64ee43d68cb9b8c3c9f23d30f317a561def29607]{
 -- Page 644, Offset 10386.
 -- 間違いなく無事では済まない。何もかも上手くいったとしても、着地の衝撃で<筋|に><肉|く>はえぐれ<骨|ほ><格|ね>は折れる。
 It's not %{i}safe%{/i} by any stretch of the imagination.
  Even if everything goes well,
- the impact of the landing will tear my muscles and break my bones.
+ the impact of the landing will tear my muscles apart, or even break my bones.
```

Comment:
this doesn't really parse right. If you want to keep this _meaning_ you probably want like
> Even if everything goes well, the impact of the landing will tear my muscles apart, and might even shatter my bones.

note that "shatter" is editorialized because "even break my bones" doesn't really read right.

That said, I'm for reverting, personally.